pom.xml

```diff
@@ -311,26 +337,26 @@
         <artifactId>commons-codec</artifactId>
         <version>1.13</version>
       </dependency>
+      <dependency>
+        <groupId>commons-logging</groupId>
+        <artifactId>commons-logging</artifactId>
+        <version>1.2</version>
+      </dependency>
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
         <version>2.7</version>
       </dependency>
       <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <version>1.2</version>
+        <groupId>io.atomix.catalyst</groupId>
+        <artifactId>catalyst-transport</artifactId>
+        <version>${catalyst.version}</version>
```

Comment:
for some of the dependencies we use a param like this, and for others we directly put a version string here. Do you think it makes sense to unify the two flavors?

Reply:
I do see the reasons to keep both flavors:
- with important deps, with set a param to indicate the version, like  `java.version`. Also sometimes we have deps of different artifacts need to share the same version, like `grpc.version` needs to be consistent across `grpc-api` and `grpc-core`.
- many other deps do not deserve a param :)